extension.js

```diff
@@ -157,6 +157,28 @@ function unsetColor(context, title) {
   }
 }
 
+async function clearAllTabColors(context) {
```

Comment:
it's better to name it clearOpenTabsColors to illustrate what it does more precisely

Reply:
the function doesn't seem to do its job correctly, it looks like closed tabs are affected as well. Try doing the following:
- open multiple tabs and colors them all
- close one of them
- execute the command
- reopen the previously closed tab
You'll notice that the tab has lost its color even if it was not part of the open tabs
the function should only target open tabs

Reply:
The functionality seems to be working as expected as you can see in the gif below, you already have a similar command to create all the colors. Were you executing that one by chance? if not please show me how you encountered the bug so I can recreate it.

[Example GIF](https://gifyu.com/image/SdCFb)

Reply:
you're right, I was executing the other command thinking it was yours, in fact your function is working all fine.

Reply:
No worries :) I will update the PR to have a better naming convention for the function!